common/models/templates/test/test-quote-behavior.js

```diff
@@ -31,7 +31,18 @@ describe('Quote behaviors', function() {
     });
 
     it.skip('RTL', function() {
```

Comment:
Is this test getting skipped?

Reply:
I left it as skipped b/c of the comment at the bottom of the `function`.  Basically, the test doesn't read intuitively, and functionally it feels about the same as the baseline LTR version.  I can un-skip it though, if y'all prefer.